runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowPipelineTranslator.java

```diff
@@ -428,7 +428,11 @@ public Job translate(List<DataflowPackage> packages) {
         environment.setDebugOptions(debugOptions);
       }
 
-      pipeline.traverseTopologically(this);
+      if (useUnifiedWorker(options)) {
```

Comment:
```suggestion
      if (DataflowRunner.useUnifiedWorker(options)) {
```

Reply:
Can we add a flag (I know we have too many) to force the  v1 translation? It could help reserve the old behavior if needed.

Reply:
If runner v2 is specified it is dead code

Reply:
(the equivalent code to this is in the Dataflow backend for v2 - this particular codepath does literally nothing for v2)

Reply:
OK.